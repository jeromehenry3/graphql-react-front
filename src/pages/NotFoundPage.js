import React from 'react';

const NotFoundPage = () => (<h1>Erreur 404 : la page que vous recherchez n'existe pas.</h1>);

export default NotFoundPage;