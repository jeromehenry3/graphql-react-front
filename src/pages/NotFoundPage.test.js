import React from 'react';
import ReactDOM from 'react-dom';
import NotFoundPage from './NotFoundPage';
import renderer from 'react-test-renderer';
import { cleanup, render } from '@testing-library/react';
import expectExport from 'expect';




// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   render(<NotFoundPage />, div);
//   ReactDOM.unmountComponentAtNode(div);
// });

// it('should render a <h1> containing : Erreur 404 : la page que vous recherchez n\'existe pas.', () => {
//     const div = document.createElement('div');
//     render(<NotFoundPage />, div);
//     expect(div).toHaveTextContent('Erreur 404 : la page que vous recherchez n\'existe pas.')
// })

test('should render <NotFoundPage /> as defined', () => {
  const component = renderer.create(
    <NotFoundPage />
  );
  let tree = component.toJSON();
  expectExport(tree).toMatchSnapshot();

  // expect(component).
});
