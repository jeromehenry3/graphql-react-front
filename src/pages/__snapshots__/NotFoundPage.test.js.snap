// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <NotFoundPage /> as defined 1`] = `
<h1>
  Erreur 404 : la page que vous recherchez n'existe pas.
</h1>
`;
